server.js

```diff
@@ -0,0 +1 @@
+const port = 8888;
```

Comment:
Your port would usually be in the `.env` file. At the moment your sever has no way to listen for requests. How can you amend this?